### PR TITLE
MP-62: add a basic root api for admins

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: src/api/root.rs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+#[derive(Deserialize)]
+pub struct RootEnforcePlusQuery {
+    pub fxa_uid: String,
+    pub enforce_plus: Option<Subscription>,
+}
+
+#[derive(Deserialize)]
+pub struct RootIsAdminQuery {
```
Comment (Guyzeroth) [inline src/api/root.rs:30]: NIT: Perhaps rename to RootSetAdminQuery

File: src/api/root.rs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    }
+    let res = root_set_is_admin(&mut conn_pool, query.into_inner());
+    if let Err(e) = res {
+        Ok(HttpResponse::BadRequest().json(format!("unable to update user: {}", e)))
+    } else {
+        Ok(HttpResponse::Created().json("updated"))
+    }
+}
+
+async fn user_by_email(
```
Comment (Guyzeroth) [inline src/api/root.rs:71]: Q: Do we really need this endpoint? It's fairly easy to ask someone to just tell us the output of /api/v1/whoami while logged in without passing an email through the ether ?

File: src/api/root.rs (hunk 1 above)
Comment (fiji-flo) [reply src/api/root.rs:30]: then maybe even RootSetIsAdminQuery? Since the field is named is_admin

File: src/api/root.rs (hunk 2 above)
Comment (Guyzeroth) [reply src/api/root.rs:71]: Ok

**Review by Guyzeroth — APPROVED** [review]

**Pull request merged**